backend/functions/lib/use-cases/attorneys.test.ts

```diff
@@ -1,58 +1,35 @@
 import { ApplicationContext } from '../adapters/types/basic';
+// TODO: investigate the duplicates of ATTORNEYS
```

Comment:
Did we do this?

Reply:
I don't recall when we observed duplicates.